ansible.properties

```diff
@@ -1,12 +1,24 @@
 category=External Analyzers
 description=Analyze Ansible playbooks.
 homepageUrl=https://github.com/sbaudoin/sonar-ansible
-archivedVersions=1.0.0,2.0.0
-publicVersions=2.1.0
+archivedVersions=1.0.0,2.0.0,2.2.0
+publicVersions=2.3.0
  
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=ansible
 
+2.3.0.description=Support for SonarQube 8
+2.3.0.date=2019-12-23
+2.3.0.sqVersions=[6.7,LATEST]
```

Comment:
Only one version can be compatible with `LATEST`

Reply:
Could you bump the start of this range up to 7.9, please? Sorry I didn't notice that the first time around.